### DISCUSSION TITLE
Stable Cascade and upscalers

### DISCUSSION THREAD

**Discussion by tscott65** [vladmandic/sdnext#3024] [Q&A]
Whenever I try to use any upscaler with SC images, I get an error "VAE not found in model". I suppose this makes sense since SC has a whole different model architcture. But I've also tried selecting a VAE manually (rather than the default "automatic") and I still get the error. 

Is there some way to use upscalers with the SC pipeline? I suppose I can try I2I with a different base model, but I'm curious what others think.

**Comment by vladmandic** [comment]
vae is necessary for latent upscale, but should not be necessary for non-latent upscale.
and definitely do not attempt to load a vae when using stable cascade.
